free-2.md

```diff
@@ -11,41 +11,58 @@ _What need is met by your idea? Who are you helping? What is that person's
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+Language for real time sports statistics
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+Someone such as a parent or coach could use their phone or computer and write a sentence such as "player 1 passed for 33 yards to player 2" the DSL would know to add 33 yards to player 1's passing total and 33 yards to player 2's catching yard total. I feel like this laguage would work best using a natural laguage syntax because it would allow for the typical user such as a parent or coach to use it without having to know a laguage. Another possible way to structure this language would be to use sports terminology such as TFL(player1, num yards) which would mean that player 1 had a tackle for a loss of num yards. For coaches and parents who understand this terminology this laguage would be very easy to infer. 
+
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+When the program runs it would create a base data structure to hold data for a particular game be that a football game or soccer game, each structure would be diffrent according to each sport. This data structure would be added to as the user utalizes the laguage, adding in individual players and their stats. After a command by the user is given, a success message can be delivered, additionally after the game has concluded the user will be able to see a represention of each individual players stats for the game as well as an entire team stats collection. This would be done by internal processing that calculates specific sports stats such as offensive yards per game. with additional features this dsl may also hav ethe ability to grab particular stats from past games for comparisions. Errors should be comunicted with error messages to the user letting them know if a particular command or stat was not stored.
+
```

Comment:
My only question is how messages are delivered, and you can see the stats. Is it purely textual? Or would there be a display window type thing.